rules_building_block/persistence_local_job_scheduling.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2023/07/11"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/11"
+
+[rule]
+author = [ "Elastic" ]
+building_block_type = "default"
+description = """
+On Linux and macOS systems, multiple methods are supported for creating pre-scheduled and periodic
+background jobs.
+"""
+index = [ "auditbeat-*", "logs-endpoint.events.*" ]
+from = "now-119m"
+interval = "60m"
+language = "eql"
+license = "Elastic License v2"
+name = "Local Job Scheduling Process"
+risk_score = 21
+rule_id = "10714bb8-c4f7-4e39-8a16-b2b7d7bc388c"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "OS: macOS", "Use Case: Threat Detection", "Tactic: Persistence", "Rule Type: BBR"]
+type = "eql"
+timestamp_override = "event.ingested"
+query = ''' process where event.type == "start" and process.name : ("cron", "at", "launchd")'''
```

Comment:
This will also generate very much noise, and does not add much useful info imo.

Reply:
Discussed in RAD removing noisy rules.